contract/Chess.sol

```diff
@@ -151,10 +151,13 @@ contract Chess is TurnBasedGame {
             throw;
         if (msg.sender == game.nextPlayer)
             throw;
-        if (game.timeoutState == 0)
-            throw;
         if (now < game.timeoutStarted + 10 minutes)
             throw;
+        if (game.timeoutState == 0)
+            game.ended = true;
+            game.winner = msg.sender;
+            GameEnded(gameId, msg.sender);
+
```

Comment:
Also, in timeout state 0 this function is not supposed to do anything! Otherwise players could claim a win without the other player ever getting a chance to react.